schema/home/results.js

```diff
@@ -18,8 +18,8 @@ const RESULTS_SIZE = 20;
 
 const moduleResults = {
   active_bids: () => [],
-  iconic_artists: () => {
-    return iconicArtists().then((artists) => {
+  trending_artists: () => {
```

Comment:
You're not wrong, but trending schmending, they're definitely 'interesting' artists!

Kidding aside, I believe (and I'm not 100% sure), that we've essentially deviated from that definition in favor of using this? It might be a mistake though, I'm really not clear.

Should we see if @briansw could refresh us as to what this should be?